tests/integrationv2/common.py

```diff
@@ -348,8 +348,10 @@ class Curves(object):
     """
     X25519 = Curve("X25519", Protocols.TLS13)
     P256 = Curve("P-256")
-    P384 = Curve("P-384")
-    P521 = Curve("P-521")
+    # providers who negotiate SSLv3 may not not send the supported
+    # groups extension so P-384 and P-521 cannot be negotiated
+    P384 = Curve("P-384", Protocols.TLS10)
+    P521 = Curve("P-521", Protocols.TLS10)
```

Comment:
nit: If this is an issue with OpenSSL and not SSLv3 itself, would it potentially make more sense for these to be disabled in the SSLv3Provider via `def supports_cipher(cls, cipher, with_curve=None)`? Maybe not really an issue though since we only have one SSLv3 provider.

Reply:
Yeah, I'm not worried about this. If we have to increase SSLv3 coverage I'll eat my hat 😉